File: Statistics.m
%Statistics
avgNumbToken = sum(metaInputs(1,:))/7204;
avgNumbTokenTrue = sum(metaInputs(1,[1:3602]))/3602;
avgNumbTokenFalse = sum(metaInputs(1,[3603:end]))/3602;

%Dá igual às stats que estão no paper

File: Statistics.py
import pandas as np
import scipy.io

# Load metaInputs from the .mat file
mat_data = scipy.io.loadmat('metaInputs.mat')

# Access the metaInputs data
metaInputs = mat_data['metaInputs']

# Statistics
avgNumbToken = np.sum(metaInputs[0, :]) / 7204
avgNumbTokenTrue = np.sum(metaInputs[0, :3602]) / 3602
avgNumbTokenFalse = np.sum(metaInputs[0, 3602:]) / 3602

# It's similar to the stats in the paper
